app/controllers/assignment_controller.rb

```diff
@@ -0,0 +1,9 @@
+class AssignmentController < ApplicationController
+  protect_from_forgery with: :exception
```

Comment:
I'd put this in `ApplicationController` or in a new controller called `AdminController` since it will only be relevant for pages that an admin handles (all the forms for creating and updating).